hat/assets/js/apps/Iaso/domains/pages/index.js

```diff
@@ -144,30 +147,37 @@ const Pages = ({ params }) => {
                                     onClick={() => {}}
                                 />
                             </a>
-                            <IconButtonComponent
-                                icon="edit"
-                                tooltipMessage={MESSAGES.edit}
-                                onClick={() =>
-                                    handleClickEditRow(
-                                        settings.row.original.slug,
-                                    )
-                                }
-                            />
-                            <IconButtonComponent
-                                icon="delete"
-                                tooltipMessage={MESSAGES.delete}
-                                onClick={() =>
-                                    handleClickDeleteRow(
-                                        settings.row.original.slug,
-                                    )
-                                }
-                            />
+                            {userHasPermission(
+                                'iaso_page_write',
```

Comment:
Since Benjamin's PR has been merged you should use constant from `hat/assets/js/apps/Iaso/utils/permissions.ts`